graphql_schema.json

```diff
@@ -11658,6 +11658,15 @@
               "isDeprecated": false,
               "deprecationReason": null
             },
+            {
+              "name": "provedState",
```

Comment:
shouldn't this be something booleanish, like `isProved` (as we have been using) or `isStateFullyProved` or `hasProvedFullState` etc.

Reply:
yeah that works for me! I named it for consistency with the OCaml type name

Reply:
hey brandon, I realized that this is called `provedState` everywhere, also in snarkyjs, not `isProved`. Ok if I leave the name for now, since the change should be one that's made across the code base?